README.md

```diff
@@ -104,6 +95,27 @@ It means:
 - alice can read data1
 - bob can write data2
 
+We also support multi-line mode by appending '\\'  in the end:  
+
+```ini
+# Matchers
+[matchers]
+m = r.sub == p.sub && r.obj == p.obj \ 
+  && r.act == p.act
+```
+
+Further more, if you are using ABAC,  you can try operator `in` like following:
```

Comment:
I like the ``in`` feature. But as Casbin is also ported to Java (jCasbin) and Node.js (Node-Casbin), I think we should remind the users that features like ``in`` are only available in the Golang Casbin.

This is because it's provided by the Golang ``govaluate`` evaluator and we use other evaluators for jCasbin and Node-Casbin. They probably don't have features like ``in``.